Sources/PackageCollections/Utility.swift

```diff
@@ -0,0 +1,74 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2020 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+ */
+
+import class Foundation.FileManager
+import TSCBasic
+
+import PackageModel
+import SourceControl
+
+// FIXME: Need a better place to put this + maybe better name
+struct Paths {
+    static func cache(fileSystem: FileSystem) -> AbsolutePath {
+        // use the idiomatic cache directory defined by FileManager when using the local file system
+        // otherwise use ~/.swift/cache
```

Comment:
```suggestion
        // otherwise use ~/.swiftpm/cache
```

Reply:
Yes, this should definitely be ~/.swiftpm/